content/rs/technology-behind-redis-enterprise.md

```diff
@@ -1,102 +1,73 @@
 ---
-Title: The Technology Behind Redis Enterprise Software (RS)
+Title: Redis Enterprise Technical Overview
 description:
 weight: 10
 alwaysopen: false
 categories: ["RS"]
 ---
-RS's unique and patented technology was developed to meet these main
-objectives:
-
-- Decouple data path from cluster management
-- Ensure consistent top performance
-- Simplify and automate operations
-
-## Decouple data path from cluster management
-
-RS's architecture completely decouples the data path from the cluster
-management in order to optimize the operations of each of these
-components.
-
-### Data path
-
-The data path is based on multiple zero-latency, multi-threaded proxies
-that reside on each of the cluster nodes to mask the underlying complexity
-of the system. This model supports multiple proxies per Redis
-database and permits the use of any regular or cluster-aware Redis
-client. This allows the user's code base to work as-is. Each proxy
-forwards client requests to the relevant Redis servers (shards). When
-new shards are added to a database, the proxy handles the new routing
-rules for the application's requests transparently, by immediately
-scaling the database performance and memory capacity without any changes
-to the application itself.
-
-The proxies also support Memcached's text and binary protocols and
-perform in-flight translation between Memcached and Redis protocol
-semantics. This allows Memcached users to enjoy many of the features
-that are not available with open source Memcached, such as built-in
-replication, auto-failover, data persistence backups and scaling
-(out/in) without losing data.
+
+Redis Enterprise was built to simplify operations while providing seamless scalability, top performance, and high availability.
```

Comment:
"was built" sounds old to me. Maybe "uses advanced technology"?

Reply:
"uses advanced technology" sounds corny and vague to me.